Makefile

```diff
@@ -73,22 +73,42 @@ install:
 
 $(BUILDDIR)/:
 	mkdir -p $@
+# The Docker image containing the generator, formatter, and linter.
+# This is generated by proto/Dockerfile. To update tools, make changes
+# there and run the Build & Push Proto Builder Image workflow.
+IMAGE := ghcr.io/tendermint/docker-build-proto:latest
+DOCKER_PROTO_BUILDER := docker run -v $(shell pwd):/workspace --workdir /workspace $(IMAGE)
+HTTPS_GIT := https://github.com/tendermint/spec.git
 
 ###############################################################################
 ###                                Protobuf                                 ###
 ###############################################################################
 
+proto-all: proto-lint proto-check-breaking
+.PHONY: proto-all
+
 proto-gen:
-	@docker pull -q tendermintdev/docker-build-proto
 	@echo "Generating Protobuf files"
-	@$(DOCKER_PROTO_BUILDER) sh ./scripts/protocgen.sh
+	@$(DOCKER_PROTO_BUILDER) buf generate --template=./buf.gen.yaml --config ./buf.yaml
```

Comment:
Yeah I think it might make sense to do this in a separate PR. I was actually hoping to distinguish between the spec protos and go-implementation-specific protos but it probably makes sense to have them generated in the same location

Reply:
Yeah, I think it might be nice to separate them out, but it'll need a bit more tinkering.